### ISSUE TITLE
Deploy to Maven Central

### ISSUE THREAD

**Issue by alwa** [gantonious/MaterialDayPicker#74]
Android Studio has started to deprecate Jcenter as it will be shut down soon. Maven central is the recommended repository to switch to.

**Comment by gantonious** [comment]
Yeah this has been brought to my attention. Luckily we still have till Feb 1, 2022 before package are no longer resolvable from jcenter. Next time a release is ready I will look into migrating to maven central.

**Comment by raghunandankavi2010** [comment]
any update on when this migration will happen?

**Comment by gantonious** [comment]
> any update on when this migration will happen?

There are currently no plans to migrate until we need to release a new version. Jfrog announced they would leave packages hosted on JCenter as read only indefinitely now: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by plexilent** [comment]
Any chances you are willing to migrate it to maven central?

**Comment by cpighini** [comment]
Any chances you are willing to migrate it to maven central?

**Comment by senpl** [comment]
Still no progress on this migration?

**Comment by cbritezm** [comment]
Is there a way to continue using this?. 
   > Could not find ca.antonious:materialdaypicker:0.7.4.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/ca/antonious/materialdaypicker/0.7.4/materialdaypicker-0.7.4.pom
       - https://jcenter.bintray.com/ca/antonious/materialdaypicker/0.7.4/materialdaypicker-0.7.4.pom
       - https://jitpack.io/ca/antonious/materialdaypicker/0.7.4/materialdaypicker-0.7.4.pom
       - https://mvnrepository.com/ca/antonious/materialdaypicker/0.7.4/materialdaypicker-0.7.4.pom
       - https://repo.maven.apache.org/maven2/ca/antonious/materialdaypicker/0.7.4/materialdaypicker-0.7.4.pom

It seems that isn't available anymore on jcenter.
